### ISSUE TITLE
how did you manage to do it in color?

### ISSUE THREAD

**Issue by flashcsgroups** [kolesa-team/newman-php-coverager#5]
добрый день . как удалось сделать в цветном варианте?
у меня отчет только в текстовом и невзрачном получается на неймане...

**Comment by WallEnd** [comment]
php ./vendor/wallend/newman-php-coverager/phpnewman --collect-reports merge ../phpnewman --html /path/to/html/report

Флаг --html позволяет сформировать отчёты в виде HTML  с красивым цветным оформлением в bootstrap теме. 

Если вы всё делаете так как указано, но у вас получаются отчеты в html но без оформления, значит у вас проблема с загрузкой статики (стили и js).

**Comment by flashcsgroups** [comment]
создается только html файл, никаких css и js не содается к нему...

**Comment by WallEnd** [comment]
> создается только html файл, никаких css и js не содается к нему...

Попробуйте проверить права на папочку, в которую вы пишете отчёт, и, если можно, приложите скрины, постараюсь воспроизвести проблему у себя.

**Comment by flashcsgroups** [comment]
a Moжет надо самому положить туда к нему файлы от бутстрапа и какие нибудь css$js ?

**Comment by WallEnd** [comment]
Нет, всё должно работать из коробки. Если есть возможность напишите мне в телеграм @WallEnd
